packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -52,12 +57,13 @@ export default function vercelServerless({
 				}
 				const outDir = getVercelOutput(config.root);
 				const viteDefine = exposeEnv(['VERCEL_ANALYTICS_ID']);
+				serverBuildDir = new URL('./dist/', config.root);
 				updateConfig({
 					outDir,
 					build: {
 						serverEntry: 'entry.mjs',
 						client: new URL('./static/', outDir),
-						server: new URL('./dist/', config.root),
+						server: serverBuildDir,
```

Comment:
Not sure if it's intentional, but the `serverBuildDir` is extracted out, but it isn't used anywhere else than here.